app/component/ImageUploader/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { typography } from 'src/vanilla-extract/typography.css';
+import styles from './styles.css';
+import { MdOutlineCameraAlt } from 'react-icons/md';
+import clsx from 'clsx';
+
+interface ImageUploaderProps {
+  maxCount: number;
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  currentCount: number;
+}
+
+export default function ImageUploader({ maxCount, onChange, currentCount }: ImageUploaderProps) {
+  const disabled = currentCount === maxCount;
```

Comment:
(제안)
조건문의 명확성을 높이기 위해서 이렇게 바꾸는 건 어떨까요?
 
```suggestion
  const disabled = currentCount >= maxCount;
```